Dockerfile.deploy

```diff
@@ -0,0 +1,4 @@
+FROM fabric8/caddy:v8759201
```

Comment:
The image tag is set to 'v8759201' here. I believe you

Reply:
Right, I should do that. I just copied over @rawlingsj's original Dockerfile without really thinking about it.